app/assets/stylesheets/config/_colors.css

```diff
@@ -0,0 +1,23 @@
+html {
+  --light-beige: #E8DBB3;
+  --yellow: #FFD96A;
+  --red: #D92A45;
+  --light-blue: #7387EB;
+  --blue: #2347EB;
+}
+
+.yellow-txt {
+  color: var(--yellow);
+}
+
+.red-txt {
+  color: var(--red);
+}
+
+.blue-txt {
+  color: var(--blue);
+}
+
+.white-txt {
+  color: white;
+}
```

Comment:
**suggestion**: I'd prefer to follow `txt-white`. That said, `txt` maybe abstracts too much. WDYT about `color-white`. Moreover, do we need these?

Reply:
I think these classes help to be more flexible when applying our texts colors. `color-white` for me reads a bit confusing, as it could be the background-color, instead of just text? Though I know color refers to the text. Maybe `txt-color-white`?

Reply:
Hm, maybe I'm just biased since I've been using this for a while. I'd be fine with just inverting the class names `txt-white` instead of `txt-yellow`.

In a sense, I like these utility classes reflecting their implementation meaning `padding-s` means `small padding property` and `color-white` means `white` `color` property.

Reply:
I changed to `color-white` and so on, it still feels weird to me, but I think it's a good way for me to get used to the fact that `color` refers to font color 🥲